qemu/tests/netperf_stress.py

```diff
@@ -67,6 +68,13 @@ def netperf_test(duration):
     for server in netperf_server:
         s_info = {}
         if server in vms:
+            smp_value = params.get_numeric("smp") or \
+                        params.get_numeric("vcpu_maxcpus")
+            if smp_value > 32:
```

Comment:
According to  the framework default rules, guest  will get half of the total number of vcpus of the host. So I think you should change the code like the following:
```
if cpu.online_count() > 2 * 32
    # since this test case tested two guests
    params["smp"] = 32
```